pylmod/base.py

```diff
@@ -80,7 +90,20 @@ def _url_format(self, service):
         return base_service_url
 
     def rest_action(self, func, url, **kwargs):
-        """Routine to do low-level REST operation, with retry"""
+        """Routine to do low-level REST operation, with retry.
+
+        Args:
+            func (callable): API function to call
+            url (str): service URL endpoint
+            kwargs (dict): addition parameters
+
+        Raises:
+            requests.RequestException
+            ValueError
+
+        Returns:
+            json
```

Comment:
I misunderstood the Python code.  I thought this line ``return response.json()`` in ``rest_action()`` returned JSON.  I'll correct the doc.